asApp/src/os/RTEMS/osdNfs.c

```diff
@@ -12,68 +12,65 @@
 /**
  * Global variables
  */
-int save_restoreNFSOK    = 1;  /* for RTEMS, NFS has been mounted before autosave starts */
-int save_restoreIoErrors = 0;  /* for accumulate the IO error numbers, when the number larger than threshold, remount NFS */
+int save_restoreNFSOK =
+    1; /* for RTEMS, NFS has been mounted before autosave starts */
+int save_restoreIoErrors = 0; /* for accumulate the IO error numbers, when the
+                                 number larger than threshold, remount NFS */
 extern volatile int save_restoreDebug;
 
 /**
  * Mount the NFS
  *
  * Pre conditions:
- *    Initialization has been done: rpcUdpInit() and nfsInit(smallPoolDepth, bigPoolDepth)
- *    Please note that when using CEXP to load the RTEMS modules, the initialization will be 
- *    done automatically
+ *    Initialization has been done: rpcUdpInit() and nfsInit(smallPoolDepth,
+ * bigPoolDepth) Please note that when using CEXP to load the RTEMS modules, the
+ * initialization will be done automatically
  *
  * Input:
- *    uidhost  - string. The NFS server and uid/gid, [<uid>'.'<gid>'@']<host>. Note better to use IP address
- *               with 'dot' notation for <host>
- *    addr     - IP address of host.  Not used here.
- *    path     - string. Absolute path on the NFS server
- *    mntpoint - string. Local path. Please note that if "mntpoint" does not exist, it will be created
+ *    uidhost  - string. The NFS server and uid/gid, [<uid>'.'<gid>'@']<host>.
+ * Note better to use IP address with 'dot' notation for <host> addr     - IP
+ * address of host.  Not used here. path     - string. Absolute path on the NFS
+ * server mntpoint - string. Local path. Please note that if "mntpoint" does not
+ * exist, it will be created
```

Comment:
Even worse than above, the input parameter documentation here has all been word-wrapped into an unreadable mess.

Reply:
As above.